Geometry/HGCalCommonData/interface/HGCalTileIndex.h

```diff
@@ -0,0 +1,17 @@
+#ifndef Geometry_HGCalCommonData_HGCalTileIndex_h
+#define Geometry_HGCalCommonData_HGCalTileIndex_h
+
+#include <cmath>
+#include <cstdint>
+
+class HGCalTileIndex {
```

Comment:
Why is this a `class` instead of a `struct`? It seems very simple. Just wondering -- no need to change it.

Reply:
I would go further and wonder why it is class at all instead of free functions in a namespace (but also I'm not suggesting to change it).